Classes/Settings.cs

```diff
@@ -490,5 +481,32 @@ protected static void SetValue(string propertyName, string value)
                 }
             }
         }
+
+        private static string GetValuePostfix(Model.Programme progInfo, string name, string defaultValue)
```

Comment:
I think these would work better as overloads for GetValue and SetValue.
Would also prefer the programme as the second or last parameter.